session-signatures/getLitActionSessionSigs/README.md

```diff
@@ -0,0 +1,24 @@
+# Getting Session Signatures using the `getLitActionSessionSigs` Method
+## Understanding the Implementation
```

Comment:
Depending on how [this comment](https://github.com/LIT-Protocol/developer-guides-code/pull/31/files#r1692119117) is resolved, this section will need to change